addons/stock/views/res_config_settings_views.xml

```diff
@@ -135,6 +135,16 @@
                                         <div class="text-muted">Use GS1 nomenclature datamatrix whenever barcodes are printed for lots and serial numbers.</div>
                                     </div>
                                 </div>
+                                <div class="row mt-2" invisible="not group_stock_production_lot or not group_stock_tracking_lot">
+                                    <span class="o_form_label">Scannable package content</span>
```

Comment:
It is not a boolean setting but if it is set, it will be used. And if it is empty, no barcode for the package contents will be printed.
For the description, maybe that ?
> If defined, a QR code containing all serial numbers contained in package will be generated, using the defined character(s) to separate each numbers

Also, now you point that, it's surely wrong to speak about serials since lots are printed too, and GS1 barcodes will be printed for every product (tracked or not) if it's applicable.
cc @thomasdeleval